tests/topotests/bgp_srv6_sid_reachability/test_bgp_srv6l3vpn_over_ipv6.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+
+#
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2023 by 6WIND
+#
+
```

Comment:
I think you forgot to rename the test file name. Shouldn't it be `test_bgp_srv6_sid_reachability.py`?

Reply:
Yes, fixed, thanks.